.github/workflows/issue_nightly_failed.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Nightly {{ workflow }} workflow of {{ date | date('YYYY-MM-DD') }} failed
+labels: area:infrastructure
```

Comment:
Which label(s), if any, should be applied to the opened issue?

Reply:
i would say itype:bug, prio:blocker